app/lib/common/widgets/member_list_entry.dart

```diff
@@ -176,17 +177,84 @@ class _ChangePowerLevelState extends State<ChangePowerLevel> {
   }
 }
 
+class _MemberListInnerSkeleton extends StatelessWidget {
+  const _MemberListInnerSkeleton();
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTile(
+      leading: Skeletonizer(
+        child: ActerAvatar(
+          mode: DisplayMode.DM,
+          avatarInfo: const AvatarInfo(
+            uniqueId: 'no id given',
+          ),
+          size: 18,
+        ),
+      ),
+      title: Skeletonizer(
+        child: Text(
+          'no id',
+          style: Theme.of(context).textTheme.bodyMedium,
+          overflow: TextOverflow.ellipsis,
+        ),
+      ),
+      subtitle: Skeletonizer(
+        child: Text(
+          'no id',
+          style: Theme.of(context)
+              .textTheme
+              .labelLarge!
+              .copyWith(color: Theme.of(context).colorScheme.neutral5),
+          overflow: TextOverflow.ellipsis,
+        ),
+      ),
+    );
+  }
+}
+
 class MemberListEntry extends ConsumerWidget {
-  final Member member;
-  final Space? space;
-  final Convo? convo;
+  final String memberId;
+  final String roomId;
   final Member? myMembership;
 
   const MemberListEntry({
     super.key,
+    required this.memberId,
+    required this.roomId,
+    this.myMembership,
```

Comment:
Why to have `myMembership` in constructer even thought we can use `final myMembership = ref.watch(roomMembershipProvider(roomId));` in this class?

Reply:
Better to have minimus argument to standalone component to make it independent of data dependency.